.github/workflows/ci.yaml

```
@@ -22,11 +22,17 @@ concurrency:
 
 jobs:
   test:
-    name: "Unit Tests and Type Checking"
+    name: "${{ matrix.os }} :: ${{ matrix.version }} :: Unit Tests and Type Checking "
     strategy:
       fail-fast: false
       matrix:
-        version: [3.11]
+        version:
+          - "3.10"
+          - "3.11"
+          - "3.12"
+
+          # Support for Python 3.13 depends on https://github.com/pytorch/pytorch/issues/130249
+          # - "3.13"
```

A more compact version could be
```suggestion
        # Support for Python 3.13 depends on https://github.com/pytorch/pytorch/issues/130249
        version: ["3.10", "3.11", "3.12"]
```